Intersect.Client/Interface/Shared/SettingsWindow.cs

```diff
@@ -425,6 +431,11 @@ void TargetingSettings_Clicked(Base sender, ClickedEventArgs arguments)
             mInterfaceSettings.Hide();
             mInformationSettings.Hide();
             mTargetingSettings.Show();
+            if (!Options.EnableAutoTurnToTarget)
+            {
+                mAutoTurnToTarget.IsChecked = false;
+                mAutoTurnToTarget.IsDisabled = true;
```

Comment:
Just make it mAutoTurnToTarget.IsDisabled = !Options......EnableAutoTurnToTarget